src/v/kafka/client/broker.cc

```diff
@@ -169,8 +169,14 @@ namespace {
 api_versions_request make_api_versions_request(api_version version) {
     api_versions_request req;
     if (version >= api_version(3)) {
+        /**
+         * In order to work with Kafka brokers this must match the following
+         * regex:
+         *
+         * [a-zA-Z0-9](?:[a-zA-Z0-9\\-.]*[a-zA-Z0-9])?
+         */
         req.data.client_software_name = "redpanda-client";
-        req.data.client_software_version = ss::sstring(redpanda_version());
+        req.data.client_software_version = "1.0.0";
```

Comment:
can we just do `redpanda_git_version()`? This way we still get our redpanda version but omit the spaces: https://github.com/redpanda-data/redpanda/blob/dev/src/v/version/version.cc.in

Reply:
i think we can, will change that